### ISSUE TITLE
clear badges

### ISSUE THREAD

**Issue by carpstriker** [peterhajas/MobileNotifier#254]
is there a way to clear badges when u reply to a text?

**Comment by Drudoo** [comment]
I have though hard on this one too. There must be an API somewhere that clears it.

**Comment by neelo** [comment]
it can be done, i know it can because bitsms has the ability to do this

**Comment by Drudoo** [comment]
Yes. So does iRealSMS but the problem is that someone needs to implement it =P

**Comment by LuisOrozco** [comment]
https://github.com/peterhajas/MobileNotifier/issues/240
